azurelinuxagent/ga/exthandlers.py

```diff
@@ -1317,19 +1333,19 @@ def update_settings(self):
 
     def create_handler_env(self):
         handler_env = {
-                "logFolder": self.get_log_dir(),
-                "configFolder": self.get_conf_dir(),
-                "statusFolder": self.get_status_dir(),
-                "heartbeatFile": self.get_heartbeat_file()
+                HandlerEnvironment.logFolder: self.get_log_dir(),
+                HandlerEnvironment.configFolder: self.get_conf_dir(),
+                HandlerEnvironment.statusFolder: self.get_status_dir(),
+                HandlerEnvironment.heartbeatFile: self.get_heartbeat_file()
             }
 
         if is_extension_telemetry_pipeline_enabled():
-            handler_env["eventsFolder"] = self.get_extension_events_dir()
+            handler_env[HandlerEnvironment.eventsFolder] = self.get_extension_events_dir()
 
         env = [{
             "name": self.ext_handler.name,
             "version": _HANDLER_ENVIRONMENT_VERSION,
```

Comment:
Why not parameterize these also?

Reply:
bump

Reply:
Ohh sorry missed this comment! I didnt do it because I thought there were already too many refactoring being done and these keys are pretty generic (name and version) but I guess they should be consistent. Let me quickly fix these up too.

Reply:
Done